pyproject.toml

```diff
@@ -2,4 +2,51 @@
 requires = ["setuptools>=56", "setuptools_scm[toml]>=3.4.1"]
```

Comment:
Let's make this `setuptools>=61.2` (which is what ini2toml generates). Presumably there's some feature in there that's needed for compatibility.

Reply:
```suggestion
requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.1"]
```